news/2024-03-06-dropping-cuda-112.md

```diff
@@ -0,0 +1,32 @@
+# Dropping CUDA 11.2 on 2024 April 22
```

Comment:
Picked `2024 April 22` as it is a Monday one full week before `2024 May 1`, which is what I understood we wanted from the core meeting. Though am somewhat flexible on this date

It does look like Passover starts that day and runs until the end of April. So Idk if we want to do this a week before that or if that would be too soon

Also if we prefer to make this the last Monday in April (even though that is only a couple days before May 1), would be ok with that too

The main thing we want to avoid is committing to maintain this in May as that runs the risk of not being able to rebuild the Docker image

Please let me know what you think

Reply:
if you would like to avoid Passover, I would make the retirement date "earlier" rather than later.

Having the CIs break with "image not found" is going to be really confusing for many.

Reply:
Our images shouldn't disappear. We just can't build new versions of them. That said, I don't even see the need to wait until April. Might as well announce it for March IMO - it's not like user feedback would change the fact that nVidia is dropping support.

Reply:
> if you would like to avoid Passover, I would make the retirement date "earlier" rather than later.

Yeah was thinking we could do April 15. Though the April 22 date doesn't really do any harm. It just gives folks more time should they need it. Work can always be done earlier (independent of the deadline)

> That said, I don't even see the need to wait until April. Might as well announce it for March IMO - it's not like user feedback would change the fact that nVidia is dropping support.

I don't know why we would provide an announcement then

In any event AIUI this announcement aligns with the expectations set at the last core meeting, which I had written down in this comment ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/5339#issuecomment-1982216827 ). Let's stick to that

Reply:
> In any event AIUI this announcement aligns with the expectations set at the last core meeting

Yeah, we were both in that meeting. No concrete dates were proposed, just "before image deprecation", and "not in a week with holidays" resp. "not on a Friday". End of March or Beginning of April would have fit that bill just as well (and your comment). 🤷